api/db/seeds/data/translations.js

```diff
@@ -0,0 +1,24 @@
+const Airtable = require('airtable');
+
+module.exports = async function(databaseBuilder) {
+  if (process.env.AIRTABLE_SAVE_TRANSLATIONS !== 'true') return;
```

Comment:
je propose le renommage suivant :
```suggestion
  if (process.env.DUPLICATE_TRANSLATIONS_TO_AIRTABLE !== 'true') return;
```